app.rb

```diff
@@ -0,0 +1,207 @@
+require_relative 'classes/book'
+require_relative 'modules/book_module'
+require_relative 'classes/label'
+require_relative 'modules/label_module'
+require_relative 'classes/game/game'
+require_relative 'modules/game_module'
+require_relative 'modules/author_module'
+require_relative 'classes/genre'
+require_relative 'classes/music_album'
+require_relative 'classes/music/load_music_genre'
+require_relative 'classes/music/save_music_genre'
+require_relative 'modules/music_album_module'
+require_relative 'modules/commands_module'
+require 'date'
+require 'colorize'
+
+class App
+  attr_accessor :music_albums, :genres
+
+  def initialize
+    initialize_collections
+    initialize_actions
+    load_books_from_json
+    GameModule.load_games_from_json(@games)
+  end
+
+  def run
+    loop do
+      render_ascii_art
+      display_menu
+      choice = gets.chomp.to_i
+      action = @actions[choice]
+      if action
+        action.call
+      else
+        puts 'Invalid choice.'
+      end
+    end
+  end
+
+  def find_label_by_title(title)
+    @labels.find { |label| label.title == title }
+  end
+
+  private
+
+  def load_books_from_json(filename = 'json/books.json')
+    return unless File.exist?(filename)
+
+    data = JSON.parse(File.read(filename))
+    data.each do |book_data|
+      book = Book.new(
+        id: book_data['id'],
+        title: book_data['title'], # Asegúrate de que esto esté en tus datos JSON
+        publisher: book_data['publisher'],
+        cover_state: book_data['cover_state'],
+        publish_date: book_data['publish_date'],
+        label: find_label_by_title(book_data['label'])
+      )
+      @books << book
+    end
+  end
```

Comment:
- [ ] I noticed that the app crashes if the `.json` files are non-existent of if they are empty. Please implement a fix that ensures the app doesn't crash the aforementioned cases 👍 
![image](https://github.com/hafiz1379/Catalog-of-my-things/assets/46839548/0ed6db8f-685b-438b-8efa-3c12d4ba9b08)